code/modules/paperwork/paper.dm

```diff
@@ -437,3 +437,52 @@
+	capable of diagnosing even the most critical, obscure or technical injuries any humanoid entity is suffering in a easy to understand format that even a non-trained health professional \
+	 can understand.<br><br>The health analyser is expected to go into full production as standard issue medical kit."
+
+/obj/item/weapon/paper/oldstat/protogun
+	name = "K14 Energy Gun Report"
+	info = "<b>*K14-Multiphase Energy Gun*</b><br><br>The K14 Prototype Energy Gun is the first Energy Rifle that has been successfully been able to not only hold a larger ammo charge \
+	than other gun models, but is capable of swapping between different energy projectile types on command with no incidents.<br><br>The weapon still suffers several drawbacks, its alternative, \
+	non laser fire mode, can only fire one round before exhausting the energy cell, the weapon also remains prohibitively expensive, nonetheless NT Market Research fully believe this weapon \
+	will form the backbone of our Energy weapon cataloge.<br><br>The K14 is expected to undergo revision to fix the ammo issues, the K15 is expected to replace the 'stun' setting with a \
+	'disable' setting in a attempt to bypass the ammo issues."
+
+/obj/item/weapon/paper/oldstat/protosing
+	name = "Singularity Generator"
+	info = "<b>*Singularity Generator*</b><br><br>Modern power generation typically comes in two forms, a Fusion Generator or a Fission Generator. Fusion provides the best space to power \
+	ratio, and is typically seen on military or high security ships and stations, however Fission reactors consume massive amounts of resources to keep operational, and are highly prone to \
+	breakdowns. Fission generators are massive and bulky, and require a large reserve of uranium to power, however they are extremely cheap to operate and oft need little maintenance once \
+	operational.<br><br>The Singularity aims to alter this, a functional Singularity is essentially a controlled Black Hole, a Black Hole that generates far more power than Fusion or Fission \
+	generators can ever hope to produce. "
+
+/obj/item/weapon/paper/oldstat/protoinv
+	name = "Laboratory Inventory"
+	info = "<b>*Inventory*</b><br><br>(1) Prototype Hardsuit<br><br>(1)Health Analyser<br><br>(1)Prototype Energy Gun<br><br>(1)Singularity Generation Disk<br><br><b>DO NOT REMOVE WITHOUT \
+	THE CAPTAIN AND RESEARCH DIRECTOR'S AUTHORISATION<b/>"
+
+/obj/item/weapon/paper/oldstat/report
+	name = "Crew Reawakening Report"
+	info = "Artifical Program's report to surviving crewmembers.<br><br>Crew were placed into cryostasis on March 10th, 2445.<br><br>Crew were awoken from cryostasis around June, 2557.<br><br> \
+	<b>SIGNIFICANT EVENTS OF NOTE</b><br>1: The primariy radioation detectors were taken offline after 112 years due to power failure, secondary radioation detectors showed no residual \
```

Comment:
"primariy" "radioation"
should be
"primary" "radiation"
radiation misspelling continues on the line below, mind you.